src/utils/include/utils/interaction/LennardJones.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef UTILS_LENNARD_JONES
+#define UTILS_LENNARD_JONES
+
+#include "utils/Vector.hpp"
+#include "utils/math/int_pow.hpp"
+#include "utils/math/sqr.hpp"
+
+namespace Utils {
+namespace Interaction {
+
+struct LennardJones {
```

Comment:
I can't see how this is a utility function, why can't we just keep it in non_bonded_interactions? If the potentials are at some point really reused, they can be their own library.